client/src/test/java/org/apache/uniffle/client/impl/ShuffleWriteClientImplTest.java

```diff
@@ -30,14 +39,88 @@
 import org.apache.uniffle.common.ShuffleBlockInfo;
 import org.apache.uniffle.common.ShuffleServerInfo;
 
+import static org.apache.uniffle.client.impl.ShuffleWriteClientImpl.waitUntilDoneOrFail;
```

Comment:
Why do we static import this?
Should the method `waitUntilDoneOrFail` be put a Util class?

Reply:
Done